amber/src/main/python/pyproject.toml

```diff
@@ -15,17 +15,13 @@
 # specific language governing permissions and limitations
 # under the License.
 
-[tool.black]
+[tool.ruff]
 line-length = 88
-target-version = ["py39", "py310", "py311", "py312"]
-extend-exclude = '''
-/(
-  proto
-)/
-'''
+target-version = "py310"
```

Comment:
I think it is better to always target the highest python version we support, in this case, python 3.12. 
@LJX2017 can you change it in a follow up PR?